hack/godep-save.sh

```diff
@@ -62,12 +62,25 @@ REQUIRED_BINS=(
   "./..."
 )
 
+# Workaround for https://github.com/tools/godep/issues/554 until: (a) it is resolved, (b) grpc adds a doc.go
```

Comment:
Not planning to merge this. It's in place to see if https://github.com/tools/godep/issues/554 is the only dependency issue in the e2e tests.